### PR TITLE
FIX: PEP8 in external

### PR DESCRIPTION
**Pull request by manu-tej**
issue #877 fixed

### CODE REVIEW

File: dipy/external/fsl.py [hunk 1]
```diff
@@ -107,12 +107,12 @@ def flirt2aff(mat, in_img, ref_img):
     *refspace* is simply *voxtrans* for the reference image.
 
     A negative determinant for the image affine is the common case, of an image
-    with a x voxel flip.  Analyze images don't store affines and flirt assumes a
-    negative determinant in these cases.
+    with a x voxel flip.  Analyze images don't store affines and flirt assumes
+    anegative determinant in these cases.
```
Comment (arokem) [inline dipy/external/fsl.py]: Space between "a" and "negative".

**Comment by arokem** [comment]
This looks OK to me, except for that one small comment.

**Comment by arokem** [comment]
All good on my side. Apologies for delay in re-review. Will give others a couple of days, before merging.

**Comment by manu-tej** [comment]
No problem

**Pull request merged**